integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/ViewPopRedirectTests/RedirectTest.java

```diff
@@ -111,16 +111,50 @@ public void testRedirect() throws Exception {
         requestHeader.put("Accept", "application/json");
 
         HttpRequestUtil.doPost(new URL(getProxyServiceURLHttp(PROXY_NAME)), inputPayload, requestHeader);
-        Thread.sleep(5000); //Make sure that the Message Processor has time to deactivate
+        //Make sure that the Message Processor has time to deactivate
+        Awaitility.await()
+                .pollInterval(2, TimeUnit.MILLISECONDS)
```

Comment:
I feel like the polling interval is too small. 2ms WDYT? Also we are waiting at most for 5 seconds and I fell that it too small too. Shall we change the interval to like 4s, since the polling interval of the message processor is 4s? And change the wait time to 2min?

Reply:
ok. That was my concern as well. Will change to 4 mills and 2 mins for all cases